module_ctx.go

```diff
@@ -271,7 +271,7 @@ func (m *baseModuleContext) OtherModuleDependencyTag(logicModule Module) Depende
 func (m *baseModuleContext) VisitDirectDeps(visit func(Module)) {
 	defer func() {
 		if r := recover(); r != nil {
-			panic(newPanicErrorf(r, "VisitDirectDeps(%s, %s, %s) for dependency %s",
+			panic(newPanicErrorf(r, "VisitDirectDeps(%s, %s) for dependency %s",
```

Comment:
Shouldn't these two be moved to the previous commit? (Though it doesn't really matter)